src/main/java/com/epam/reportportal/junit5/ReportPortalExtension.java

```diff
@@ -327,30 +338,24 @@ private void startTestItem(ExtensionContext context, List<Object> arguments, Str
 		idMapping.put(context.getUniqueId(), itemId);
 	}
 
-	private String getCodeRef(Method method) {
-		return method.getDeclaringClass().getCanonicalName() + "." + method.getName();
-	}
-
 	private Set<ItemAttributesRQ> getAttributes(Method method) {
 		return ofNullable(method.getAnnotation(Attributes.class)).map(AttributeParser::retrieveAttributes).orElseGet(Sets::newHashSet);
-
 	}
 
 	private TestCaseIdEntry getTestCaseId(String codeRef) {
 		return new TestCaseIdEntry(codeRef, codeRef.hashCode());
 	}
 
-	@Nullable
 	private TestCaseIdEntry getTestCaseId(Method method, String codeRef, List<Object> arguments) {
-		return ofNullable(method.getAnnotation(TestCaseId.class)).map(testCaseId -> {
-			if (testCaseId.parametrized()) {
-				return TestCaseIdUtils.getParameterizedTestCaseId(method, arguments);
+		if (method != null) {
+			TestCaseId tcId = method.getAnnotation(TestCaseId.class);
```

Comment:
Probably, `testCaseId` or `caseId` looks better

Reply:
What the problem?

Reply:
OK